core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -241,6 +245,7 @@ public void complete() {
     private void closeListener(Status newStatus) {
       if (!listenerClosed) {
         listenerClosed = true;
+        onStreamDeallocated();
```

Comment:
You mean in `AbstractClientStream2.TransportState`? Sort of. The difference is `listener` is a different type (client vs server).